flexmeasures/data/schemas/scheduling/shiftable_load.py

```diff
@@ -0,0 +1,118 @@
+from __future__ import annotations
+
+from datetime import datetime
+import pytz
+import pandas as pd
+
+from marshmallow import (
+    Schema,
+    post_load,
+    fields,
+    pre_load,
+)
+
+from flexmeasures.data.models.time_series import Sensor
+from flexmeasures.data.schemas.times import AwareDateTimeField, DurationField
+
+
+from flexmeasures.data.schemas.sensors import SensorIdField
+
+from enum import Enum
+
+
+class LoadType(Enum):
+    INFLEXIBLE = "INFLEXIBLE"
+    BREAKABLE = "BREAKABLE"
+    SHIFTABLE = "SHIFTABLE"
+
+
+class OptimizationSense(Enum):
+    MAX = "MAX"
+    MIN = "MIN"
+
+
+class TimeIntervalSchema(Schema):
+    start = AwareDateTimeField(required=True)
+    duration = DurationField(required=True)
+
+
+class ShiftableLoadFlexModelSchema(Schema):
+    cost_sensor = SensorIdField(data_key="cost-sensor")
```

Comment:
I believe this should be replaced by using the `FlexContextSchema.consumption_price_sensor` field.

Reply:
Now I realized why I called this sensor `price_sensor`: this sensor is used as the optimization signal, it can be energy costs, CO2 emissions, etc... However, I refactored the code to use `consumption_price_sensor` to be more consistent with the rest. Shall I remove it from the FlexModel and keep it in the flex context as usual?

Reply:
Yes.
In the end, we can discuss that the context should use different names, e.g. consumption_cost_sensor and production_reward_sensor, to he more generic.